x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/upgrade_prebuilt_rules_table/upgrade_prebuilt_rules_table_context.tsx

```diff
@@ -272,21 +260,31 @@ export const UpgradePrebuiltRulesTableContextProvider = ({
     selectedRules,
     loadingRules,
     dataUpdatedAt,
-    flyoutRule,
-    isFlyoutInstallButtonDisabled,
+    canPreviewedRuleBeUpgraded,
```

Comment:
This state is not used outside the flyout, so we don't need to expose it. Let's keep it internal.